tests/validation_tool/icmp.rflx

```diff
@@ -0,0 +1,118 @@
+package ICMP is
```

Comment:
>Why isn't the real specification fixed?

Because of this error: 
```
[gw4] [ 27%] FAILED tests/test_specs.py::test_spec[icmp.rflx] 

Parsing icmp.rflx
Processing ICMP
icmp.rflx:116:23: generator: error: unsupported checksum (consider --ignore-unsupported-checksum option)
```
Should the `--ignore-unsupported-checksum` argument be used in the test? Or is there another way to resolve this?

Reply:
Yes, this flag should be used in the test as long as https://github.com/Componolit/RecordFlux/issues/276 has not been solved.

Reply:
Fixed in 7ac482ae, ICMP and IPv4 specs updated in 8c0da0c3, 02ea89cf, 7ac482ae